### DISCUSSION TITLE
Conceptualizing GeneIntegrationMatrix data

### DISCUSSION THREAD

**Discussion by dgagler** [GreenleafLab/ArchR#2053] [Questions / Documentation]
Hi, looking for some clarification on how to think about the data in the GeneIntegrationMatrix slot after integrating RNA and ATAC data from the same samples. Particularly in contrast to Gene Scores. I understand Gene Scores to be gene expression estimates based on the open chromatin...but is the GeneIntegrationMatrix data essentially just an import of the RNA gene expression data, assigned to each ATAC cell?

**Comment by rcorces** [comment]
correct

**Comment by dgagler** [comment]
Got it, thanks.
